Gemfile.lock

```diff
@@ -927,4 +929,4 @@ RUBY VERSION
    ruby 3.0.3p157
 
 BUNDLED WITH
-   2.4.3
+   2.4.5
```

Comment:
As far as I can see Dependabot doesn't support managing this yet. I'm curious why you have a later version actually? 

I'm not sure if we have any particular policy, but I guess this normally only gets updated as a result of updating Ruby. 

Maybe we should update more frequently. I can see there's one vulnerability mentioned on the changelog, which is fixed in `2.4.11`